### PR TITLE
Volume IO Tests CAS-505 CAS-506

### PR DESCRIPTION
**Pull request by blaisedias**
Remove redundant test in nvmf_vol

### CODE REVIEW

**Review by chriswldenyer — COMMENTED** [review]
There is a problem with leaking PVCs on failure, which Jan also mentioned. My tests also have this issue. It also prevents any subsequent test creating a PVC with the same name if it still exists.

**Comment by blaisedias** [comment]
> There is a problem with leaking PVCs on failure, which Jan also mentioned. My tests also have this issue. It also prevents any subsequent test creating a PVC with the same name if it still exists.

I have fixed this now.

File: test/e2e/basic_volume_io/basic_volume_io_test.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
+// JIRA: CAS-505
+// JIRA: CAS-506
```
Comment (paulyoong) [inline test/e2e/basic_volume_io/basic_volume_io_test.go:2]: I think we should remove this?

**Review by paulyoong — APPROVED** [review]

File: test/e2e/basic_volume_io/basic_volume_io_test.go (hunk 1 above)
Comment (blaisedias) [reply test/e2e/basic_volume_io/basic_volume_io_test.go:2]: There was some discussion early on in which it was decided that we add this so for ease of tracking test results.

File: test/e2e/basic_volume_io/basic_volume_io_test.go [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	Eventually(func() bool {
+		return common.IsPodRunning(fioPodName)
+	},
+		defTimeoutSecs,
+		"1s",
+	).Should(Equal(true))
+
+	// Run the fio test
+	common.RunFio(fioPodName, 20)
+	podNames = podNames[:len(podNames)-1]
```
Comment (chriswldenyer) [inline test/e2e/basic_volume_io/basic_volume_io_test.go:54]: Shouldn't you do this after DeletePod() has succeeded?

**Review by jkryl — APPROVED** [review]

**Comment by blaisedias** [comment]
bors try

**Review by chriswldenyer — APPROVED** [review]

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**